providers/common/provider_err.c

```diff
@@ -111,6 +111,8 @@ static const ERR_STRING_DATA PROV_str_reasons[] = {
     {ERR_PACK(ERR_LIB_PROV, 0, PROV_R_KEY_SIZE_TOO_SMALL),
     "key size too small"},
     {ERR_PACK(ERR_LIB_PROV, 0, PROV_R_LENGTH_TOO_LARGE), "length too large"},
+    {ERR_PACK(ERR_LIB_PROV, 0, PROV_R_MISMATCHING_SHARED_PARAMETERS),
```

Comment:
_shared_ or _domain_?  Not sure which is better.

Reply:
Yeah I had that and then changed it - Domain parameters applies to ec also.. Do you want it renamed?

Reply:
We use domain parameters elsewhere. Please do.